### PR TITLE
move relative humidity measurement cluster to seperate xml file

### PR DESCRIPTION
**Pull request by ghost**
#### Problem
https://github.com/project-chip/connectedhomeip/pull/7224 moved the temperature measurement cluster to a separate xml file. This PR does the same for the relative humidity measurement cluster. This PR also addresses a similar issue as https://github.com/project-chip/connectedhomeip/pull/7810. It sets spec complaint defaults to min / max measured values.

#### Change overview
- move relative humidity measurement cluster to separate xml file
- set spec compliant min / max measured values default values

#### Testing
Locally tested with zap.

### CODE REVIEW

**Review by Damian-Nordic — APPROVED** [review]

File: src/app/zap-templates/zcl/relative-humidity-measurement-cluster.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <code>0x0405</code>
+    <define>RELATIVE_HUMIDITY_MEASUREMENT_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" tickFrequency="half" init="false">true</server>
+    <attribute side="server" code="0x0000" define="RELATIVE_HUMIDITY_MEASURED_VALUE" type="INT16U" writable="false" reportable="true" optional="false">measured value</attribute>
+    <attribute side="server" code="0x0001" define="RELATIVE_HUMIDITY_MIN_MEASURED_VALUE" type="INT16U" min="0x0000" max="0x270F" writable="false" default="0xFFFF" optional="false">min measured value</attribute>
+    <attribute side="server" code="0x0002" define="RELATIVE_HUMIDITY_MAX_MEASURED_VALUE" type="INT16U" min="0x0001" max="0x2710" writable="false" default="0xFFFF" optional="false">max measured value</attribute>
+    <attribute side="server" code="0x0003" define="RELATIVE_HUMIDITY_TOLERANCE" type="INT16U" min="0x0000" max="0x0800" writable="false" reportable="true" optional="true">tolerance</attribute>
+  </cluster>
+</configurator>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/relative-humidity-measurement-cluster.xml:32]: Add newline at end of file, please?

**Review by bzbarsky-apple — APPROVED** [review]
The 0xFFFF default doesn't really make much sense to me, but ok...

**Comment by eve-cxrp** [comment]
> The 0xFFFF default doesn't really make much sense to me, but ok...

Currently there is

**Comment by woody-apple** [comment]
@saurabhst @msandstedt @LuDuda @andy31415 ?

**Review by msandstedt — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
> So 0xFFFF indicates that the value is not defined

That's not how Matter represents "not defined", is the point.

**Comment by eve-cxrp** [comment]
> That's not how Matter represents "not defined", is the point.

Well, since matter has no spec for relative humidity measurement yet. This is the best we can do for now. At least in my opinion.

**Comment by bzbarsky-apple** [comment]
>  This is the best we can do for now

This is why I marked approved and said "but ok", yes?

What we _should_ do is have an issue tracking this.

**Pull request merged**